great_tables/_gt_data.py

```diff
@@ -610,7 +611,7 @@ def order_groups(self, group_order: RowGroups):
         # TODO: validate
         return self.__class__(self.rows, self.group_rows.reorder(group_order))
 
-    def group_indices_map(self) -> list[tuple[int, str | None]]:
+    def group_indices_map(self) -> list[tuple[int, GroupRowInfo | None]]:
```

Comment:
now returning GroupRowInfo directly, since we need the group ids, not just their defaulted labels to match group name locs